src/services/encryptPassword.js

```diff
@@ -0,0 +1,10 @@
+const bcrypt = require('bcrypt');
+const saltRounds = 10;
+
+module.exports.encryptPassword = async function (password) {
+    var pass;
+    await bcrypt.hash(password, saltRounds).then(function(hash) {
+        pass = hash;
+    });
+    return pass;
+};
```

Comment:
Add a new line at the end of the file
```suggestion
};

```